ibm/mas_devops/playbooks/oneclick_add_visualinspection.yml

```diff
@@ -25,8 +33,30 @@
           - lookup('env', 'MAS_CONFIG_DIR') != ""
           - lookup('env', 'IBM_ENTITLEMENT_KEY') != ""
         fail_msg: "One or more required environment variables are not defined"
-
+    - include_role:
+      # Configure Object Storage
+        name: ibm.mas_devops.cos
+      when:
+        - mas_app_settings_visualinspection_object_storage_enabled
+        - configure_cos
+        - cos_type in ['ibm','ocs']
+    - include_role:
+      # Configure COS bucket in IBM Cloud Object Storage (10 min)
+        name: ibm.mas_devops.cos_bucket
+      when:
+        - mas_app_settings_visualinspection_object_storage_enabled
+        - mas_app_settings_visualinspection_object_storage_workspace
+        - configure_cos_bucket
+        - cos_type in ['ibm','aws']
```

Comment:
just wondering why we're checking different cos types , `['ibm', 'aws']` here vs `['ibm,'cos']` in the previous role?

Reply:
With this we are allowing options for users to also provision object storage if they don't have one setup yet.
While using `cos` role, user will be able to configure either `ocs` or `ibm cloud object storage` instances to be used.
Whereas while using the `cos_bucket` role, users will be able to create a bucket within the ibm cloud object storage instance or in aws/s3 (in aws, users can allocate a bucket directly without the need of a COS instance per say). 

If `cos_type` is different from these two options, then either user is using `ocs` which don't required a bucket to be created, or won't need a bucket at all to be created using the automation i.e user might have one bucket already configured.